hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/TestHoodieDeltaStreamerSchemaEvolutionBase.java

```diff
@@ -155,6 +156,8 @@ protected HoodieDeltaStreamer.Config getDeltaStreamerConfig(String[] transformer
 
   protected HoodieDeltaStreamer.Config getDeltaStreamerConfig(String[] transformerClasses, boolean nullForDeletedCols,
                                                               TypedProperties extraProps) throws IOException {
+    extraProps.setProperty(HoodieMetadataConfig.ENABLE_METADATA_INDEX_PARTITION_STATS.key(),"false");
```

Comment:
[HUDI-8587](https://issues.apache.org/jira/browse/HUDI-8587) tracks schema evolution fix for partition stats